bin/download

```diff
@@ -36,11 +36,7 @@ download_golang () {
 
     platform=$(get_platform)
     arch=$(get_arch)
-    if [ -z "${RTX_GOLANG_DOWNLOAD_MIRROR:-}" ]; then
```

Comment:
💭 this check was reversed. It could've been changed to `-n`, but it's simpler to use the variable default syntax